### ISSUE TITLE
In ionic 4 cordova project, after updating  plugin from version 2.0.0 to 2.0.1 android build fails

### ISSUE THREAD

**Issue by pakbaz** [Azure/azure-mobile-apps-cordova-client#57]
I get this error:
```
BUILD FAILED
 in 8s
17 actionable tasks: 4 executed, 13 up-to-date
(node:80736) UnhandledPromiseRejectionWarning: Error: /Users/sepehr/Repos/cp-frontend/platforms/android/gradlew: Command failed with exit code 1 Error output:
/Users/sepehr/Repos/cp-frontend/platforms/android/AndroidManifest.xml:12:73-109 Error:
	Attribute meta-data#com.primerica.azure_poc.azure_mobileapps_redirect_uri_scheme@value at AndroidManifest.xml:12:73-109 requires a placeholder substitution but no value for <redirectUriScheme> is provided.
/Users/sepehr/Repos/cp-frontend/platforms/android/AndroidManifest.xml:18:56-93 Error:
	Attribute data@scheme at AndroidManifest.xml:18:56-93 requires a placeholder substitution but no value for <redirectUriScheme> is provided.
/Users/sepehr/Repos/cp-frontend/platforms/android/AndroidManifest.xml Error:
	Validation failed, exiting

FAILURE: Build failed with an exception.
```

When I roll back to version 2.0.0 problem resolves.

**Comment by jaelim-ms** [comment]
Thanks for reporting the issue. We are looking at the issue now, will get back to you soon.

**Comment by ruslan-bikkinin** [comment]
Hi @pakbaz 
New version contains fix for Google authorization due to it is no longer allow OAuth requests to Google in InAppBrowser plugin so it was replaced with CustomTabs implementation from [azure-mobile-apps-android-client](https://github.com/Azure/azure-mobile-apps-android-client/pull/84). All you need to configure is to add these lines to `android/build.gradle`:

```
android {
     ...

    buildTypes {
        ...

        debug {
            manifestPlaceholders = ['redirectUriScheme': "APP_URI_SCHEME"]
        }

        release {
            manifestPlaceholders = ['redirectUriScheme': "APP_URI_SCHEME"]
        }

        ...
    }
}
```
where `APP_URI_SCHEME` is uri scheme of your application for [deep-link](https://developer.android.com/training/app-links/deep-linking.html).

Please let us know if it helps you.

**Comment by ruslan-bikkinin** [comment]
Hi @parthnayak7448 I prepared PR https://github.com/Azure/azure-mobile-apps-cordova-client/pull/59 with fix which will be merged with a pending release soon.

**Comment by parthnayak7448** [comment]
Hello, @ruslan-bikkinin  thanks for the reply. But I would like to inform that I am facing this error `com.microsoft.azure:azure-mobile-android:3.3.0' depends on one or more Android Libraries but is a jar` 
whenever I installed and try to run in android. 

Can you please help me step by step to solve this issue. I am trying to solve this since last week.

Thanks.

**Comment by dhei** [comment]
We merged #59 and released v2.0.2 today. This issue should be resolved and can be closed.
